packages/tinybench-plugin/src/index.ts

```diff
@@ -22,6 +23,7 @@ export function withCodSpeed(bench: Bench): Bench {
   if (codspeedRunnerMode === "disabled") {
     return bench;
   }
+  setupCore();
```

Comment:
What's the benefit of calling `setupCore` here instead? That means the setup is done before actually registering all the benches